#! /bin/sh /usr/share/dpatch/dpatch-run
## cups-1.5.0-compatibility.dpatch by Steve Langasek <steve.langasek@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Directly include cups/ppd.h header, for compatibility with CUPS 1.5.0
## DP: release.

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' hplip-3.11.7~/prnt/hpps/hppsfilter.c hplip-3.11.7/prnt/hpps/hppsfilter.c
--- hplip-3.11.7~/prnt/hpps/hppsfilter.c	2011-07-24 18:58:05.000000000 +0000
+++ hplip-3.11.7/prnt/hpps/hppsfilter.c	2011-08-09 20:51:54.150214165 +0000
@@ -38,6 +38,7 @@
 #include <unistd.h>
 #include <fcntl.h>
 #include <cups/cups.h>
+#include <cups/ppd.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 
